src/components/Request/FormFields/PaySystem.tsx

```diff
@@ -0,0 +1,61 @@
+import { Combobox, Label, Option, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { DropdownIcon } from "@fluentui/react-icons-mdl2";
+import { PAYSYSTEMS } from "consts/PaySystems";
+
+const PaySystem = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <Label
+        id={name + "Id"}
+        size="small"
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+      >
+        <DropdownIcon className="requestFieldIcon" />
+        Pay System
+      </Label>
+      <Controller
+        name="paySystem"
+        control={form.control}
```

Comment:
You should add a rule to make this field required.  The user can manually clear it out -- not that they would do this :)